python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -13,46 +26,38 @@
 from dagster._core.storage.pipeline_run import RunRecord, RunsFilter
 from dagster._core.storage.tags import TagType, get_tag_type
 from dagster._legacy import DagsterRunStatus, PipelineDefinition
-from graphene import ResolveInfo
 
 from .external import ensure_valid_config, get_external_pipeline_or_raise
-from .utils import UserFacingGraphQLError, capture_error
+from .utils import capture_error
 
 if TYPE_CHECKING:
-    from ..schema.asset_graph import GrapheneAssetNode
-    from ..schema.util import HasContext
-
-
-def is_config_valid(pipeline_def, run_config, mode):
-    check.str_param(mode, "mode")
-    check.inst_param(pipeline_def, "pipeline_def", PipelineDefinition)
-
-    run_config_schema = create_run_config_schema(pipeline_def, mode)
-    validated_config = validate_config(run_config_schema.config_type, run_config)
-    return validated_config.success
+    from dagster_graphql.schema.tags import GraphenePipelineTagAndValues
 
+    from ..schema.asset_graph import GrapheneAssetLatestInfo, GrapheneAssetNode
+    from ..schema.errors import GrapheneRunNotFoundError
+    from ..schema.execution import GrapheneExecutionPlan
+    from ..schema.logs.events import GrapheneRunStepStats
+    from ..schema.pipelines.config import GraphenePipelineConfigValidationValid
+    from ..schema.pipelines.pipeline import GrapheneEventConnection, GrapheneRun
+    from ..schema.pipelines.pipeline_run_stats import GrapheneRunStatsSnapshot
+    from ..schema.runs import GrapheneRunGroup
+    from ..schema.util import HasContext
 
-def get_validated_config(pipeline_def, run_config, mode):
-    from ..schema.pipelines.config import GrapheneRunConfigValidationInvalid
 
+def is_config_valid(
+    pipeline_def: PipelineDefinition, run_config: Mapping[str, Any], mode: str
+) -> bool:
     check.str_param(mode, "mode")
     check.inst_param(pipeline_def, "pipeline_def", PipelineDefinition)
 
     run_config_schema = create_run_config_schema(pipeline_def, mode)
-
     validated_config = validate_config(run_config_schema.config_type, run_config)
-
-    if not validated_config.success:
-        raise UserFacingGraphQLError(
-            GrapheneRunConfigValidationInvalid.for_validation_errors(
-                pipeline_def.get_external_pipeline(), validated_config.errors
-            )
-        )
-
-    return validated_config
+    return bool(validated_config.success)
```

Comment:
How is this not a meaningful behavior change?

Reply:
That diff is misleading because that's actually comparing the return of a different function (`validate_config`), which was deleted entirely because it was never called-- `is_config_valid` returned `validation_config.success`, which is `bool` or `None`. But on second look `is_config_valid` should also be removed entirely because it is also never called-- looks like an old config validation pathway? In any case I updated the diff to remove it entirely.

Reply:
Ah yes removing that would be much more clear.

Reply:
It's removed-- do you have other requested changes